File: Carnaval/src/app/Models/ocorrencia.ts
import { Atendimento } from './atendimento';
import { Doenca } from './doenca';

export class Ocorrencia {
  private _atendimento: Atendimento;

  public get Atendimento(): Atendimento {
    return this._atendimento;
  }

  public set Atendimento(value: Atendimento) {
    this._atendimento = value;
  }

  private _doencas: Doenca[];

  public get Doencas(): Doenca[] {
    return this._doencas;
  }

  public set Doencas(value: Doenca[]) {
    this._doencas = value;
  }


}

File: SOM.API/Models/Ocorrencia.cs
using SOM.OR;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SOM.API.Models
{
    [Serializable]
    public class Ocorrencia
    {
        private Atendimento _atendimento;
        private IList<Doenca> _doencas;

        public virtual Atendimento Atendimento
        {
            get { return _atendimento; }
            set { _atendimento = value; }
        }
        public virtual IList<Doenca> Doencas
        {
            get { return _doencas; }
            set { _doencas = value; }
        }
    }
}